src/Config/ConfigFileWatcher.cs

```diff
@@ -31,14 +38,19 @@ private void OnConfigFileChange(object sender, FileSystemEventArgs e)
     {
         try
         {
-            if (_configProvider is null)
+            if (_configLoader is null)
```

Comment:
summary comment needs completion for params.

also nit in the summary: `this trigger function` -> `this event handler`